library.properties

```diff
@@ -0,0 +1,9 @@
+name=MemoryFree
+version=1.0
+author=Various; see https://forum.arduino.cc/index.php?topic=27536.0
+maintainer=Neil McNeight (http://mcneight.org/)
+sentence=Functions for calculating amount for free data memory (RAM).
+paragraph=Functions for calculating amount for free data memory (RAM).
```

Comment:
The `paragraph` value is typically displayed immediately after the `sentence` value, so duplicating the `sentence` content in `paragraph` has a harmful effect. It should either contain additional unique text or else be left blank.

Reply:
I will amend to add further detail.

Reply:
I have made this change